app/articles_download.py

```diff
@@ -0,0 +1,44 @@
+import requests
+from newschatbot.app.config import FEED_URL
+from newschatbot.app.model import Article, db, Questions, Answers
+import xmltodict
+
+# maybe how to use in serverless? https://stackoverflow.com/questions/61960140/serverless-how-to-periodically-run-a-flask-command-in-a-aws-lambda-function
+
+
+def download_data_from_url():
+    try:
+        response = requests.get(FEED_URL)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        raise SystemExit(err)
+    data = xmltodict.parse(response.content)
+    return data
+
+
+def update_articles():
+    articles = download_data_from_url()
+    for key, value in articles.items():
+        order = 0
+        for i, j in value.items():
+            for q in j:
```

Comment:
Mohli bysme to ještě upravit na trochu deskriptivnější názvy? V tomhle místě už iterujeme přes `key`, `value`, `i`, `j` a `q` a není hned jasné, čemu ty hodnoty odpovídají